xml/decoders/lenz/accelDecel.xml

```diff
@@ -55,6 +55,8 @@
     <comment xml:lang="de">Wertebereich 0-255</comment>
     <label xml:lang="cs">Zrychlení</label>
     <comment xml:lang="cs">Meze 0-255</comment>
+    <label xml:lang="cs">Inèrcia d'Acceleració</label>
+    <comment xml:lang="cs">Rang de 0-255</comment>
```

Comment:
I suspect that this should have been `xml:lang="ca"` given that `cs` is the code for Czech.